clients/R/tests/testthat/setup.R

```diff
@@ -1,25 +1,26 @@
-options(warnPartialMatchDollar = TRUE, warn = 2)
-options(warnPartialMatchArgs = TRUE, warn = 2)
-options(warnPartialMatchAttr = TRUE, warn = 2)
-
-stan_folder <- file.path("..", "..", "..", "..", "test_models")
-
-bernoulli_model <- tinystan::StanModel$new(file.path(stan_folder, "bernoulli", "bernoulli_model.so"))
+options(warnPartialMatchDollar = TRUE)
+options(warnPartialMatchArgs = TRUE)
+options(warnPartialMatchAttr = TRUE)
+options(warn = 0)
+stan_folder <- file.path("..", "test_models")
+print(getwd())
+bernoulli_model <- tinystan_model(file.path(stan_folder, "bernoulli", "bernoulli.stan"))
+tinystan:::api_version(bernoulli_model)
 BERNOULLI_DATA <- "{\"N\": 10, \"y\": [0, 1, 0, 0, 0, 0, 0, 0, 0, 1]}"
 
-gaussian_model <- tinystan::StanModel$new(file.path(stan_folder, "gaussian", "gaussian_model.so"))
+gaussian_model <- tinystan_model(file.path(stan_folder, "gaussian", "gaussian.stan"))
 
-empty_model <- tinystan::StanModel$new(file.path(stan_folder, "empty", "empty_model.so"))
-multimodal_model <- tinystan::StanModel$new(file.path(stan_folder, "multimodal",
-    "multimodal_model.so"))
-simple_jacobian_model <- tinystan::StanModel$new(file.path(stan_folder, "simple_jacobian",
-    "simple_jacobian_model.so"))
+empty_model <- tinystan_model(file.path(stan_folder, "empty", "empty.stan"))
+multimodal_model <- tinystan_model(file.path(stan_folder, "multimodal", "multimodal.stan"))
+simple_jacobian_model <- tinystan_model(file.path(stan_folder, "simple_jacobian",
+  "simple_jacobian.stan"))
```

Comment:
These were intentionally using the `_model.so` suffix to make local testing and CI both faster when you're not interested in directly testing the compilation feature

Reply:
Sorry I'm confused, there's no .so file in the folders up in top level test examples. Do these sit on jenkins?

Reply:
The files are built once for each operating system and then re-used by the different languages/versions. This also allows caching across different runs if only the interface code is changing, not the C++ sources. 

You can run `make test_models` to do the same thing locally

Reply:
Okay